servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultClassHttpDeserializer.java

```diff
@@ -64,7 +64,7 @@ public T deserialize(final HttpHeaders headers, final Buffer payload) {
     private void checkContentType(final HttpHeaders headers) {
         if (!checkContentType.test(headers)) {
             throw new SerializationException("Unexpected headers, can not deserialize. Headers: "
-                    + headers.toString());
+                    + headers.toString(HeaderUtils.DEFAULT_DEBUG_HEADER_FILTER));
```

Comment:
nit: please use static import for `DEFAULT_DEBUG_HEADER_FILTER` to improve readability.
(here and other places where you are using this constant.)